### PR TITLE
README.md

### PR DESCRIPTION
**Pull request by ganatrask**
The environment can be well setup with Ubuntu 18.04 & ROS melodic.

### CODE REVIEW

**Comment by jmplaza** [comment]
Hi @iamshyamganatra ,

you are right. Watch out the spelling of Melodic in your pull-request. Anyway, I will merge your corrected pull request after you have successfully tested ROS Melodic with one of the current operating exercises: [https://jderobot.github.io/RoboticsAcademy/portfolio/follow_line](https://jderobot.github.io/RoboticsAcademy/portfolio/follow_line) or [https://jderobot.github.io/RoboticsAcademy/portfolio/obstacle_avoidance/ ](https://jderobot.github.io/RoboticsAcademy/portfolio/obstacle_avoidance/ ) 

Did you check them with ROS Melodic? You don't need to solve them, just to check that they are properly launched using ROS Melodic.

Cheers!

**Comment by mohamedsayed18** [comment]
I tried the two examples and they worked 
OS: Ubuntu 18.04 
ROS: ROS melodic

**Comment by mohamedsayed18** [comment]
[Readme]in step three:
https://github.com/mohamedsayed18/RoboticsAcademy#step-three-add-jderobot-sources
this is step is also required for Ubuntu 18.04

**Comment by igarag** [comment]
With the update of the installation as well as many of the exercises it is possible that this issue has already been resolved. Can you review it with the [new guide](https://jderobot.github.io/RoboticsAcademy/installation/)?

**Comment by iamshyamganatra** [comment]
I tried with the updated installation guide, 
there are some minor changes that I felt to be made, I tried looking on GitHub for the updated guide but can't find can you share me the file where I can suggest changes?

**Comment by igarag** [comment]
The file containing the installation guide is in this path.

https://github.com/JdeRobot/RoboticsAcademy/blob/master/docs/_pages/installation.md

File: README.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ There are exercises about drone programming, about computer vision, about mobile
 
 ### Software Infraestructure: Ubuntu/Debian
 
-The programming environment is composed of the (a) Gazebo simulator, (b) ROS middleware and (c) the Academy package. All this software is open source so there are alternative ways to install all of them directly from the source code. Currently we use Gazebo-7.4.0, ROS Kinetic and JdeRobot-Academy (2018-06-06) releases. Follow the next steps to have the environment up and running, ready to use.
+The programming environment is composed of the (a) Gazebo simulator, (b) ROS middleware and (c) the Academy package. All this software is open source so there are alternative ways to install all of them directly from the source code. Currently we use Gazebo-7.4.0, ROS Medloci and JdeRobot-Academy with (2018-06-06) releases Ubuntu 18.04.3 . Follow the next steps to have the environment up and running, ready to use.
```
Comment (nkhedekar) [inline README.md]:
```suggestion
The programming environment is composed of the (a) Gazebo simulator, (b) ROS middleware and (c) the Academy package. All this software is open source so there are alternative ways to install all of them directly from the source code. Currently we use Gazebo-7.4.0, ROS Melodic and JdeRobot-Academy with (2018-06-06) releases Ubuntu 18.04.3 . Follow the next steps to have the environment up and running, ready to use.
```

**Pull request closed without merging**